src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/CanAccessCohort/CanAccessCohortHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.CommitmentsV2.Data;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Application.Queries.CanAccessCohort
+{
+    public class CanAccessCohortHandler : IRequestHandler<CanAccessCohortQuery, bool>
```

Comment:
Do you think it's worth unit testing this class?

Reply:
Definitely, and this task is in the backlog